docs/Makefile.am

```diff
@@ -1,5 +1,8 @@
 
-html_DATA = $(srcdir)/html/*.*
+html_DATA = $(builddir)/html/*.*
+
+$(html_DATA):
```

Comment:
Is this going to work? I'm not sure if the in-git version did (does Automake handle wildcards?), but I don't see how could it handle the wildcard in `builddir` correctly…

I *think* the goal here was to have `make install` install the docs, and that it won't anymore. That's probably why `bootstrap` built the docs too.

The HTML build certainly should be included in the dist tarball (which I think still works).

Reply:
It seems to work, but I'm not really sure why/how myself, so I've changed the makefile to use a Doxygen tag file as a dependency, this seems to be simpler and more clear.